server_admin/topics/user-federation/ldap.adoc

```diff
@@ -195,3 +195,38 @@ must be enabled explicitly in the LDAP configuration page. See the documentation
 
 WARNING: Always verify that user passwords are properly hashed and not stored as plaintext by inspecting a changed
 directory entry using `ldapsearch` and base64 decode the `userPassword` attribute value.
+
+[[_ldap_troubleshooting]]
+==== Troubleshooting
+
+- It is useful to increase the logging level to TRACE for the category `org.keycloak.storage.ldap` . This can be done in the logging
+subsystem in the `standalone(-ha).xml` file. With this setting, there is big amount
+of logging messages sent to the `server.log` file in the `TRACE` level, including the logging for all queries to the LDAP server and the parameters, which were
```

Comment:
Change "there is big amount of logging messages sent"
to
"many logging messages are sent"